README.md

```diff
@@ -1,78 +1,151 @@
-  * Contribute to the product design with the product team
-  * Dispatch & manage support team issue
-  * Involve in the hiring process
-
-## Other roles
+This job ladder exposes all roles inside IWD dev team, and define the goals, expectations and responsibilities of each role.
+To climb this ladder you will know what to do: Constant progression, being involved and meeting your individual objectives.
+
+However, some implicit expectations are not specified in the roles:
+
+- Being involved into code reviews
+- Communicating, seeking help when stuck
+- Being a "team player"
+
+## Track
+
+| Individual Contributor (IC) | Manager           |
+| --------------------------- | ----------------- |
+| Engineer I                  |                   |
+| Engineer II / OPS           |                   |
+| Senior Engineer I / OPS     |                   |
+| Senior Engineer II / OPS    |                   |
+| Lead Engineer               |                   |
+|                             | Engineer Director |
+|                             | CTO               |
+
+## Individual Contributor
+
+### Engineer I
+
+- Experience: ~0–2 years
+- Report: Engineering Director
+- Goals:
+  - Works on features, with guidance
+  - Manages bugs & hotfixes
+  - Writes tests, with guidance
+  - Participate to review teammates work
+  - Focus on growing as an engineer developer
+  - Learn one aspect of the technical stack
+  - Learn to deliver on-time
+
+### Engineer II
+
+- Experience: ~2–4 years
+- Report: Engineering Director
+- Goals:
+  - Work on features, with minimal guidance
+  - Manage bugs & hotfixes
+  - Write tests, with minimal guidance
+  - Review & Approve teammates work
+  - Master one aspect of his project technical stack
+  - Deliver on-time
+  - Learn domain / business skills
+
+### Senior Engineer I
+
+- Experience: ~4–7 years
+- Report: Engineering Director
+- Goals:
+  - Work on epics, with minimal guidance
+  - Work on features, without guidance
+  - Write tests, without guidance
+  - Review & Approve teammates work
+  - Master his project technical stack
+  - Expert in one technology
+  - Expert in one large area of the codebase
+  - Mentor Engineer I
+  - Focus on developing domain / business skills
+
+### Senior Engineer II
+
+- Experience: ~5–10 years
+- Report: Engineering Director
+- Goals:
+  - Work on epics, without guidance
+  - Work on features, without guidance
+  - Write tests, without guidance
+  - Review & Approve teammates work
+  - Master his project technical stack
+  - Expert in more than one technology
+  - Expert in many areas of the codebase
+  - Mentor Engineer II & Senior Engineer I
+  - Take initiatives to find and qualify structural issues
+  - Manage technical backlog with Lead Engineer
+  - Know, Define, Challenge, Evangelize best practices, processes and documentation
+  - Expert in domain / business skills
+
+### Lead Engineer
```

Comment:
You should also introduce the transversality of the role, maybe: Can manage cross-team project